include/dynamic-graph/CMakeLists.txt

```diff
@@ -48,8 +48,6 @@ SET(${PROJECT_NAME}_HEADERS
   all-commands.h
 
   logger.h
-    )
+)
 
-INSTALL(FILES ${${PROJECT_NAME}_HEADERS}
-  DESTINATION include/${PROJECT_NAME}
-  )
+SET(PUBLIC_HEADER ${${PROJECT_NAME}_HEADERS})
```

Comment:
I think you have moved all of this in the main CMakeLists.txt, so this should be removed from the main one or from this one.

I think it would be better to define this list here, but I'm not sure yet how this would work with the CMake package.

Reply:
Fixed